### PR TITLE
Add Smorgasbord post

### PR DESCRIPTION
**Pull request by mblue9**

### CODE REVIEW

**Comment by mblue9** [comment]
@almahmoud I made a few edits, is this ok by you? feel free to make any other edits

File: posts/2023-06-16-smorgasbord-recap/index.qmd [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  - name: Alex Mahmoud
+    url: https://github.com/almahmoud
+date: 2023-06-16
+categories:
+  - events
+  - education
+image: "bioconductor-galaxy.png"
+---
+
+We're delighted to share that Bioconductor was part of the [Galaxy Smorgasbord](https://gallantries.github.io/video-library/events/smorgasbord3/) global online training for the first time this year. Over the week of May 22-26, we extended the power of R and Bioconductor to a wider, global audience.
```
Comment (hexylena) [inline posts/2023-06-16-smorgasbord-recap/index.qmd:17]:
```suggestion
We're delighted to share that Bioconductor was part of the [Galaxy Smörgåsbord](https://gallantries.github.io/video-library/events/smorgasbord3/) global online training for the first time this year. Over the week of May 22-26, we extended the power of R and Bioconductor to a wider, global audience.
```

:grimacing: sorry for the terrible event name

File: posts/2023-06-16-smorgasbord-recap/index.qmd [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Competition
+
+To further encourage active participation, we ran a friendly competition. Participants who completed a Bioconductor tutorial and submitted an R history before June 4th were eligible to win a free virtual registration to the Bioconductor conference [BioC2023](https://bioc2023.bioconductor.org/). We congratulate the winner, Paul Yu, a Master's student from De La Salle University, Manila, Philippines!
+
+## Our New Infrastructure
+
+The Smorgasbord event also marked the public launch of the new Bioconductor Workshop platform, a successor to the [Orchestra platform](https://github.com/seandavi/Orchestra). The new platform ([workshop.bioconductor.org](https://workshop.bioconductor.org/)) is a Kubernetes deployment of a Galaxy fork, notably leveraging the Kubernetes Interactive Tools runner to launch Bioconductor RStudio-based containers, each representing a workshop. This stack is running on a cluster of Jetstream2 nodes, supported by NSF ACCESS allocation BIR190004. This platform will be employed for the annual Bioconductor conference [BioC2023](https://bioc2023.bioconductor.org/), and the [ISMB/ECCB 2023](https://www.iscb.org/ismbeccb2023) conference.
```
Comment (hexylena) [inline posts/2023-06-16-smorgasbord-recap/index.qmd:29]:
```suggestion
The Smörgåsbord event also marked the public launch of the new Bioconductor Workshop platform, a successor to the [Orchestra platform](https://github.com/seandavi/Orchestra). The new platform ([workshop.bioconductor.org](https://workshop.bioconductor.org/)) is a Kubernetes deployment of a Galaxy fork, notably leveraging the Kubernetes Interactive Tools runner to launch Bioconductor RStudio-based containers, each representing a workshop. This stack is running on a cluster of Jetstream2 nodes, supported by NSF ACCESS allocation BIR190004. This platform will be employed for the annual Bioconductor conference [BioC2023](https://bioc2023.bioconductor.org/), and the [ISMB/ECCB 2023](https://www.iscb.org/ismbeccb2023) conference.
```

File: posts/2023-06-16-smorgasbord-recap/index.qmd [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+![Bioconductor Galaxy Platform](bioconductor-galaxy.png)
+
+## Looking Ahead
+
+By adopting Galaxy, Bioconductor hopes to broaden both the breadth of workshops available in the Galaxy Training Network, as well as the audience of Bioconductor workshops. A broader range of learning opportunities will empower scientists to excel in their research and tackle more complex questions with greater confidence and efficiency. We also hope to work with the usegalaxy.* administrators to make Bioconductor workshops available on all usegalaxy instances. We look forward to seeing more learners engage with Bioconductor tutorials, and we are excited about the potential for Bioconductor to contribute to more global learning initiatives like this.
```
Comment (hexylena) [inline posts/2023-06-16-smorgasbord-recap/index.qmd:35]: > We also hope to work with the usegalaxy.* administrators to make Bioconductor workshops available on all usegalaxy instances

yes! I was just discussing this with some other admins, would be really nice to get all of these containers upstreamed to galaxy's codebase, then they'll be available on every galaxy, and probably most of the .* instances will enable them.

File: posts/2023-06-16-smorgasbord-recap/index.qmd [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+
+## Acknowledgements
+
+A big thank you to all [the Bioconductor Smorgasbord instructors](https://gallantries.github.io/video-library/modules/bioconductor), Laurent Gatto, Charlotte Soneson, Kevin Rue-Albrecht, Federico Marini, Stefano Mangiola, Aidan Lakshman, Chloe Mirzayi, Maddy Griswold, Megan Vandenberg, Stephanie Zimmerman, Matt Ritchie, Yunshun Chen, Dharmesh Bhuva, Mike Love, Alan Murphy, all the helpers, including our [Bioconductor Carpentries Instructors](https://bioconductor.github.io/biocblog/posts/2023-02-24-carpentries-update/), and the Smorgasbord organizers, Helena Rasche and Saskia Hiltemann.
```
Comment (hexylena) [inline posts/2023-06-16-smorgasbord-recap/index.qmd:40]:
```suggestion
A big thank you to all [the Bioconductor Smörgåsbord instructors](https://gallantries.github.io/video-library/modules/bioconductor), Laurent Gatto, Charlotte Soneson, Kevin Rue-Albrecht, Federico Marini, Stefano Mangiola, Aidan Lakshman, Chloe Mirzayi, Maddy Griswold, Megan Vandenberg, Stephanie Zimmerman, Matt Ritchie, Yunshun Chen, Dharmesh Bhuva, Mike Love, Alan Murphy, all the helpers, including our [Bioconductor Carpentries Instructors](https://bioconductor.github.io/biocblog/posts/2023-02-24-carpentries-update/), and the Smorgasbord organizers, Saskia Hiltemann and Helena Rasche.
```

she should come first, she's been the main powerhouse behind this event.

**Review by hexylena — COMMENTED** [review]
awesome!

File: posts/2023-06-16-smorgasbord-recap/index.qmd (hunk 1 above)
Comment (mblue9) [reply posts/2023-06-16-smorgasbord-recap/index.qmd:17]: 😄 thanks for fixing

File: posts/2023-06-16-smorgasbord-recap/index.qmd (hunk 3 above)
Comment (mblue9) [reply posts/2023-06-16-smorgasbord-recap/index.qmd:35]: that would be great!

**Pull request merged**